src/program.ts

```diff
@@ -3628,17 +3628,16 @@ export class Local extends VariableLikeElement {
     /** Declaration reference. */
     declaration: VariableLikeDeclarationStatement = parent.program.makeNativeVariableDeclaration(name)
   ) {
-    super(
-      ElementKind.Local,
-      name,
-      parent,
-      declaration
-    );
+    super(ElementKind.Local, name, parent, declaration);
```

Comment:
Why include this in the diff?
```suggestion
    super(
      ElementKind.Local,
      name,
      parent,
      declaration
    );
```